server/storage/database.ts

```diff
@@ -11,6 +11,7 @@ const isSSLDisabled = env.PGSSLMODE === "disable";
 const poolMax = env.DATABASE_CONNECTION_POOL_MAX ?? 5;
 const poolMin = env.DATABASE_CONNECTION_POOL_MIN ?? 0;
 const url = env.DATABASE_CONNECTION_POOL_URL || env.DATABASE_URL;
+const schema = env.DATABASE_SCHEMA ?? null;
```

Comment:
```suggestion
const schema = env.DATABASE_SCHEMA;
```

Seems like types are complaining that `undefined` is expected here